src/responsibleai/rai_analyse/create_rai_insights.py

```diff
@@ -92,11 +96,18 @@ def create_constructor_arg_dict(args):
     class_names = get_from_args(
         args, "classes", custom_parser=json_empty_is_none_parser, allow_none=True
     )
+    feature_metadata_dict = get_from_args(
+        args, "feature_metadata", custom_parser=json.loads, allow_none=True
+    )
+    feature_metadata = FeatureMetadata()
+    if 'dropped_features' in feature_metadata_dict.keys():
```

Comment:
what about other fields like 'identity_feature', 'categorical_features'? Why are we skipping those?

Reply:
This is the first step to add feature_metadata, we can add other fields like 'identity_feature', 'categorical_features' later

Reply:
The 'identity_feature' is already working in OSS. Why can't we add the 'identity_feature' to this PR?

Reply:
sounds good, will add 'identity_feature' to this PR as well!